tests/Agent/IntegrationTests/ContainerIntegrationTests/LinuxKafkaTests.cs

```diff
@@ -27,8 +27,8 @@ protected LinuxKafkaTest(T fixture, ITestOutputHelper output) : base(fixture)
         _fixture.TestLogger = output;
 
         _topicName = GenerateTopic();
-        var brokerName = "broker" + _topicName;
-        ((ContainerApplication)_fixture.RemoteApplication).DockerDependencies.Add(brokerName);
+        var brokerName = "kafka-broker";
+        //((ContainerApplication)_fixture.RemoteApplication).DockerDependencies.Add(brokerName);
```

Comment:
Can we remove the commented out bits if we don't need them any more?

Reply:
Yeah, I missed that one...